cloud/shared/validate_variable_definitions.py

```diff
@@ -28,7 +29,9 @@ def load_repo_variable_definitions_files(self):
         # add their paths here.
         cwd = os.getcwd()
         definition_file_paths = [
-            cwd + "/cloud/shared/variable_definitions.json"
+            f"{cwd}/cloud/shared/variable_definitions.json",
```

Comment:
nit: single quotes?

Reply:
I agree that I prefer single-quotes. I left this as double-quotes for consistency with the rest of the file. It'd be nice to do a pass on all of our Python files in a single PR and convert to preferring one over the other, but probably not super high priority.